manon/components/button-cta.scss

```diff
@@ -0,0 +1,130 @@
+@use "../mixins/apply-from-theme";
+@use "../variables" as theme;
+
+button,
+a.button,
+input[type="button"],
+input[type="submit"],
+input[type="reset"] {
+  &.cta {
+    @include apply-from-theme.apply-from-theme(
+      (
+        justify-content: theme.$button-cta-justify-content,
+        gap: theme.$button-cta-gap,
+        background-color: theme.$button-cta-background-color,
+        color: theme.$button-cta-text-color,
+        font-family: theme.$button-cta-font-family,
+        font-size: theme.$button-cta-font-size,
+        font-weight: theme.$button-cta-font-weight,
+        line-height: theme.$button-cta-line-height,
+        text-decoration: theme.$button-cta-text-decoration,
+        text-align: theme.$button-cta-text-align,
+        padding-top: theme.$button-cta-padding-top,
+        padding-right: theme.$button-cta-padding-right,
+        padding-bottom: theme.$button-cta-padding-bottom,
+        padding-left: theme.$button-cta-padding-left,
```

Comment:
Some styling seem to have disappeared. For example: 
line 31 and 32 of mixins/button.scss 

 margin: 0;
 box-sizing: border-box;
 
 While these shouldn't be set as optional for themes they should be included in the default styling of the button.